fastlane/README.md

```diff
@@ -0,0 +1,70 @@
+fastlane documentation
```

Comment:
⛏️ No need to track this.

(I thought we had a `skip_docs` in our `Fastfile` but I'm not finding it, we'll address that in another PR)

Reply:
Fair, we can remove this file then.